src/constants/mockData/resultPanel.tsx

```diff
@@ -0,0 +1,18 @@
+// Mock data for "ResultPanel" component
+export const primaryBtnHandler = jest.fn();
+export const secondaryBtnHandler = jest.fn();
```

Comment:
Fix the extension of all the files. If there is no need of `.tsx` extension, then convert it to `.ts`.